### PR TITLE
Updated to new accessRights

### PR DESCRIPTION
**Pull request by sondrev**

### CODE REVIEW

File: publication-commons/src/test/java/no/unit/nva/publication/permission/strategy/PublicationPermissionStrategyTest.java [hunk 1]
```diff
@@ -259,12 +261,12 @@ void shouldGivePermissionToOperateOnPublicationWhenEditor() throws JsonProcessin
...
 
     private static Function<AccessRight, String> getCognitoGroup(URI institutionId) {
-        return accessRight -> accessRight.name() + AT + institutionId.toString();
+        return accessRight -> accessRight.toPersistedString() + AT + institutionId.toString();
     }
 
     private List<AccessRight> getCuratorWithPublishDegreeAccessRight() {
         var curatorAccessRight = getCuratorAccessRights();
-        curatorAccessRight.add(AccessRight.PUBLISH_DEGREE);
+        curatorAccessRight.add(MANAGE_PUBLISHING_REQUESTS);
```
Comment (truhacevkir) [inline publication-commons/src/test/java/no/unit/nva/publication/permission/strategy/PublicationPermissionStrategyTest.java:269]: MANAGE_PUBLISHING_REQUESTS er gamle PUBLISH_DEGREE?

**Review by truhacevkir — APPROVED** [review]
En kommentar ang navnet på den nye AccessRight, det er sikkert rett :)

File: publication-commons/src/main/java/no/unit/nva/publication/permission/strategy/CuratorPermissionStrategy.java [hunk 2]
```diff
@@ -14,10 +14,10 @@ public boolean hasPermission(RequestInfo requestInfo, Publication publication) {
             return false;
         }
         if (isDegree(publication)) {
-            return hasAccessRight(requestInfo, PUBLISH_DEGREE);
+            return hasAccessRight(requestInfo, MANAGE_PUBLISHING_REQUESTS);
```
Comment (monaullah) [inline publication-commons/src/main/java/no/unit/nva/publication/permission/strategy/CuratorPermissionStrategy.java:17]: I think you have mapped `PUBLISH_DEGREE` to the wrong new access right here

File: publication-commons/src/main/java/no/unit/nva/publication/permission/strategy/CuratorPermissionStrategy.java [hunk 3]
```diff
@@ -14,10 +14,10 @@ public boolean hasPermission(RequestInfo requestInfo, Publication publication) {
...
         }
 
-        return hasAccessRight(requestInfo, EDIT_OWN_INSTITUTION_RESOURCES);
+        return hasAccessRight(requestInfo, MANAGE_RESOURCES_STANDARD);
```
Comment (monaullah) [inline publication-commons/src/main/java/no/unit/nva/publication/permission/strategy/CuratorPermissionStrategy.java:20]: Should this be mapped to `MANAGE_OWN_RESOURCES`? (Not sure)

File: publication-commons/src/main/java/no/unit/nva/publication/permission/strategy/CuratorPermissionStrategy.java (hunk 2 above)
Comment (monaullah) [reply publication-commons/src/main/java/no/unit/nva/publication/permission/strategy/CuratorPermissionStrategy.java:17]: I think the correct one is `MANAGE_DEGREE`?

File: publication-commons/src/main/java/no/unit/nva/publication/permission/strategy/EditorPermissionStrategy.java [hunk 4]
```diff
@@ -21,20 +21,20 @@ public static EditorPermissionStrategy fromRequestInfo(RequestInfo requestInfo)
     @Override
     public boolean hasPermission(RequestInfo requestInfo, Publication publication) {
         if (isDegree(publication)) {
-            return hasAccessRight(requestInfo, PUBLISH_DEGREE)
-                   && hasAccessRight(requestInfo, EDIT_ALL_NON_DEGREE_RESOURCES);
+            return hasAccessRight(requestInfo, MANAGE_PUBLISHING_REQUESTS)
```
Comment (monaullah) [inline publication-commons/src/main/java/no/unit/nva/publication/permission/strategy/EditorPermissionStrategy.java:24]: `MANAGE_DEGREE`?

File: publication-commons/src/main/java/no/unit/nva/publication/permission/strategy/EditorPermissionStrategy.java [hunk 5]
```diff
@@ -21,20 +21,20 @@ public static EditorPermissionStrategy fromRequestInfo(RequestInfo requestInfo)
...
     public boolean hasPermission(Publication publication) {
         return hasPermissionToOperateOnDegree(publication)
-               || hasAccessRight(requestInfo, EDIT_ALL_NON_DEGREE_RESOURCES);
+               || hasAccessRight(requestInfo, MANAGE_RESOURCES_ALL);
     }
 
     private boolean hasPermissionToOperateOnDegree(Publication publication) {
-        return isDegree(publication) && hasAccessRight(requestInfo, PUBLISH_DEGREE)
-               && hasAccessRight(requestInfo, EDIT_ALL_NON_DEGREE_RESOURCES);
+        return isDegree(publication) && hasAccessRight(requestInfo, MANAGE_PUBLISHING_REQUESTS)
```
Comment (monaullah) [inline publication-commons/src/main/java/no/unit/nva/publication/permission/strategy/EditorPermissionStrategy.java:37]: `PUBLISH_DEGREE`?

File: publication-commons/src/main/java/no/unit/nva/publication/permission/strategy/EditorPermissionStrategy.java [hunk 6]
```diff
@@ -21,20 +21,20 @@ public static EditorPermissionStrategy fromRequestInfo(RequestInfo requestInfo)
...
+               && hasAccessRight(requestInfo, MANAGE_RESOURCES_ALL);
```
Comment (monaullah) [inline publication-commons/src/main/java/no/unit/nva/publication/permission/strategy/EditorPermissionStrategy.java:38]: Is `EDIT_ALL_NON_DEGREE_RESOURCES` mapped to `MANAGE_RESOURCES_ALL`?

File: publication-commons/src/test/java/no/unit/nva/publication/permission/strategy/PublicationPermissionStrategyTest.java (hunk 1 above)
Comment (monaullah) [reply publication-commons/src/test/java/no/unit/nva/publication/permission/strategy/PublicationPermissionStrategyTest.java:269]: Think this should be mapped to `MANAGE_DEGREE`?

File: publication-commons/src/test/java/no/unit/nva/publication/permission/strategy/PublicationPermissionStrategyTest.java [hunk 7]
```diff
@@ -307,14 +309,14 @@ private Publication createPublicationWithContributor(String contributorName, URI
 
     private List<AccessRight> getEditorAccessRights() {
         var accessRights = new ArrayList<AccessRight>();
-        accessRights.add(AccessRight.PUBLISH_DEGREE);
-        accessRights.add(AccessRight.EDIT_ALL_NON_DEGREE_RESOURCES);
+        accessRights.add(MANAGE_PUBLISHING_REQUESTS);
```
Comment (monaullah) [inline publication-commons/src/test/java/no/unit/nva/publication/permission/strategy/PublicationPermissionStrategyTest.java:312]: Same as previous comments

File: publication-commons/src/test/java/no/unit/nva/publication/permission/strategy/PublicationPermissionStrategyTest.java [hunk 8]
```diff
@@ -307,14 +309,14 @@ private Publication createPublicationWithContributor(String contributorName, URI
...
+        accessRights.add(MANAGE_RESOURCES_ALL);
         return accessRights;
     }
 
     private List<AccessRight> getCuratorAccessRights() {
         var accessRights = new ArrayList<AccessRight>();
-        accessRights.add(AccessRight.EDIT_OWN_INSTITUTION_RESOURCES);
+        accessRights.add(AccessRight.MANAGE_RESOURCES_STANDARD);
```
Comment (monaullah) [inline publication-commons/src/test/java/no/unit/nva/publication/permission/strategy/PublicationPermissionStrategyTest.java:319]: `MANAGE_OWN_RESOURCES`?

File: publication-rest/src/main/java/no/unit/nva/publication/create/CreatePublicationHandler.java [hunk 9]
```diff
@@ -126,7 +126,7 @@ private void setLocationHeader(SortableIdentifier identifier) {
 
     private boolean isThesisAndHasNoRightsToPublishThesAndIsNotExternalClient(CreatePublicationRequest request, RequestInfo requestInfo) {
 
-        return isThesis(request) && !requestInfo.userIsAuthorized(PUBLISH_DEGREE.name()) && !requestInfo.clientIsThirdParty();
+        return isThesis(request) && !requestInfo.userIsAuthorized(MANAGE_PUBLISHING_REQUESTS) && !requestInfo.clientIsThirdParty();
```
Comment (monaullah) [inline publication-rest/src/main/java/no/unit/nva/publication/create/CreatePublicationHandler.java:129]: Same as previous comments

File: publication-rest/src/main/java/no/unit/nva/publication/update/UpdatePublicationHandler.java [hunk 10]
```diff
@@ -248,7 +248,7 @@ private boolean userIsPublicationOwner(UserInstance userInstance, Publication pu
     }
 
     private boolean userUnauthorizedToPublishThesisAndIsNotExternalClient(RequestInfo requestInfo) {
-        return !requestInfo.userIsAuthorized(PUBLISH_DEGREE.name()) && !requestInfo.clientIsThirdParty();
+        return !requestInfo.userIsAuthorized(MANAGE_PUBLISHING_REQUESTS) && !requestInfo.clientIsThirdParty();
```
Comment (monaullah) [inline publication-rest/src/main/java/no/unit/nva/publication/update/UpdatePublicationHandler.java:251]: Same as previous comments

File: publication-rest/src/test/java/no/unit/nva/publication/create/CreatePublicationHandlerTest.java [hunk 11]
```diff
@@ -379,7 +379,7 @@ private InputStream createPublicationRequest(CreatePublicationRequest request) t
             .withCurrentCustomer(testOrgId)
             .withTopLevelCristinOrgId(topLevelCristinOrgId)
             .withBody(request)
-            .withAccessRights(testOrgId, EDIT_OWN_INSTITUTION_RESOURCES.name(), PUBLISH_DEGREE.name())
+            .withAccessRights(testOrgId, MANAGE_RESOURCES_STANDARD, MANAGE_PUBLISHING_REQUESTS)
```
Comment (monaullah) [inline publication-rest/src/test/java/no/unit/nva/publication/create/CreatePublicationHandlerTest.java:382]: Same as previous comments

File: publication-rest/src/test/java/no/unit/nva/publication/create/CreatePublicationHandlerTest.java [hunk 12]
```diff
@@ -400,7 +400,7 @@ private InputStream createPublicationRequestFromString(String request) throws Js
...
```
Comment (monaullah) [inline publication-rest/src/test/java/no/unit/nva/publication/create/CreatePublicationHandlerTest.java:403]: Same as previous comments

File: publication-rest/src/test/java/no/unit/nva/publication/delete/DeletePublicationHandlerTest.java [hunk 13]
```diff
@@ -432,7 +432,7 @@ void shouldUpdateDeletedResourceWithDuplicateOfValueWhenResourceIsADuplicate()
         var request = createRequestWithDuplicateOfValue(publication.getIdentifier(),
                                                         randomString(),
                                                         publication.getPublisher().getId(),
-                                                        PUBLISH_DEGREE,
+                                                        MANAGE_PUBLISHING_REQUESTS,
```
Comment (monaullah) [inline publication-rest/src/test/java/no/unit/nva/publication/delete/DeletePublicationHandlerTest.java:435]: Same as previous comments

File: publication-rest/src/test/java/no/unit/nva/publication/delete/DeletePublicationHandlerTest.java [hunk 14]
```diff
@@ -402,7 +402,7 @@ void shouldReturnSuccessWhenEditorUnpublishesDegree() throws ApiGatewayException
 
         var publisherUri = publication.getPublisher().getId();
         var inputStream = createHandlerRequest(publication.getIdentifier(), randomString(), publisherUri,
-                                               PUBLISH_DEGREE);
+                                               MANAGE_PUBLISHING_REQUESTS);
```
Comment (monaullah) [inline publication-rest/src/test/java/no/unit/nva/publication/delete/DeletePublicationHandlerTest.java:405]: Same as previous comments

File: publication-rest/src/test/java/no/unit/nva/publication/delete/DeletePublicationHandlerTest.java [hunk 15]
```diff
@@ -493,7 +493,7 @@ void shouldPersistUnpublishRequestWhenDeletingPublishedPublication()
         var publication = createAndPersistDegreeWithoutDoi();
         publicationService.publishPublication(UserInstance.fromPublication(publication), publication.getIdentifier());
         var publisherUri = publication.getPublisher().getId();
-        var request = createHandlerRequest(publication.getIdentifier(), randomString(), publisherUri, PUBLISH_DEGREE);
+        var request = createHandlerRequest(publication.getIdentifier(), randomString(), publisherUri, MANAGE_PUBLISHING_REQUESTS);
```
Comment (monaullah) [inline publication-rest/src/test/java/no/unit/nva/publication/delete/DeletePublicationHandlerTest.java:496]: Same as previous comments

File: publication-rest/src/test/java/no/unit/nva/publication/update/UpdatePublicationHandlerTest.java [hunk 16]
```diff
@@ -798,7 +798,7 @@ private InputStream userWithAccessRightToEditDegree(Publication publicationUpdat
                 .withPathParameters(pathParameters)
                 .withCurrentCustomer(customerId)
                 .withBody(publicationUpdate)
-                .withAccessRights(customerId, PUBLISH_DEGREE.name(), PUBLISH_DEGREE.name())
+                .withAccessRights(customerId, MANAGE_PUBLISHING_REQUESTS)
```
Comment (monaullah) [inline publication-rest/src/test/java/no/unit/nva/publication/update/UpdatePublicationHandlerTest.java:801]: Same as previous comments

**Review by monaullah — CHANGES_REQUESTED** [review]

**Review by anetteOlli — APPROVED** [review]

**Pull request closed without merging**